operators/pkg/controller/elasticsearch/sset/list.go

```diff
@@ -81,6 +83,22 @@ func (l StatefulSetList) GetActualPods(c k8s.Client) ([]corev1.Pod, error) {
 	return allPods, nil
 }
 
+// MatchActualPods returns true if actual existing pods match what is specified in the StatefulSetList.
+// It may return false if there are pods in the process of being created (but not created yet)
+// or terminated (but not removed yet).
+func (l StatefulSetList) MatchActualPods(c k8s.Client, es v1alpha1.Elasticsearch) (bool, error) {
```

Comment:
Isn't it more `MatchExpectedPods`?
Trying to find another name: `ArePodsReconciled`, `IsPodsReconcilationDone` or just `IsReconciled`.

Reply:
I agree, that's confusing 👍 
I think I like `IsPodsReconcilationDone`, maybe simplified slightly to `.PodReconciliationDone()` (singular form).